scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+# sleep
+# This sleep is the sum of the time when docker is pulled, started,
+# and the height of the node increases to 1 or more.
+sleep 30
+
+# send transaction to node
+cur_path=`pwd`
+chain_dir=${cur_path}"/finschia/template/.finschia"
+artifacts_path=${cur_path}"/../artifacts/"
```

Comment:
I think it is better to do the test all in the docker.

Reply:
I want to be able to check it on local, so I'll leave it as is for now.

Reply:
>do the test all in the docker 

After considering it, I fixed it as pointed out in this commit.

[fix: change to send fnsad tx wasm execute ~ on docker instead of on…](https://github.com/Finschia/finschia-wasm/pull/3/commits/19b863ef9e0e09e959529308fbcba3b4568e2ef8)